stl/inc/type_traits

```diff
@@ -1473,6 +1473,14 @@ _NODISCARD constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_c
     return _STD move(_Arg);
 }
 
+// Currently, invoke() isn’t depicted as constexpr in the Standard, and implementers are forbidden from “strengthening”

```

Comment:
You have smart quotes of doom here. I think CI will block this.

Reply:
All quotes should be valid in c++.

«don't at me»